keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/ec2.kt

```diff
@@ -15,7 +15,5 @@
  */
 package com.netflix.spinnaker.keel.ec2
 
-import com.netflix.spinnaker.keel.SPINNAKER_API_V1
-
-const val SPINNAKER_EC2_API_V1 = "ec2.$SPINNAKER_API_V1"
+const val SPINNAKER_EC2_API_V1 = "ec2/v1"
```

Comment:
So this previously resulted in `ec2.spinnaker.netflix.com/v1`. Do we really want to get rid of the "FQDN" there?

Reply:
I felt like it didn't add any value

Reply:
Fair enough. No issues.